user-guide/location-marker-manager.rst

```diff
@@ -0,0 +1,56 @@
+Location Marker Manager
+=======================
+
+The Location Marker manager can be accessed from the menu by selecting **Tools
+> Location Marker Manager**. It allows for the management of location markers'
+positions, icons, and names.
+
+.. image:: images/location-marker-manager-01.png
+
+Adding a location marker
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The user can add a location marker by clicking on the Add button in the
+location marker manager or by using the Add Location Marker hotkey while on the
+map (the default is M). The Add button requires the manual input of a latitude
+and longitude, whereas the hotkey will place the marker at the location of the
+mouse on the map. Both methods will show the `Edit Location Marker dialog
+<#edit-location-marker-dialog>`_. When adding a new location marker, canceling
+the dialog will delete the location marker.
+
+Editing a location marker
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A location marker can be edited by either right clicking it on the map, or
+double clicking it in the Location Marker Manager. Either will open the `Edit
+Location Marker dialog <#edit-location-marker-dialog>`_. When editing,
+canceling the dialog will not apply any changes to the location marker.
+
+Removing a location marker
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A location marker can be removed by either selecting it in the Location Marker
+manager and pressing Remove or by pressing Delete in the `Edit Location Marker
+dialog <#edit-location-marker-dialog>`_.
+
+Edit Location Marker Dialog
```

Comment:
For some reason, providing the link text as you have it above will give this section an id of `id3`. Suggest giving this an explicit name right above this, and updating the links (edit-location-marker-dialog doesn't work!?).

https://www.sphinx-doc.org/en/master/usage/referencing.html#cross-referencing-arbitrary-locations